cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/config/SecurityContext.java

```diff
@@ -31,7 +31,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .and().authorizeRequests()
                 // Anyone can access the urls
                 .antMatchers("/", "/resources/**", "/login", "/loginFailed", "/register", "/location/**", "/paper/**",
-                        "/stats", "/affiliation/**", "/arxiv/**", "/logout", "/reset/**", "/search", "/affiliations","/publications")
+                        "/stats", "/affiliation/**", "/arxiv/**", "/logout", "/reset/**", "/{searchType}/search",
```

Comment:
`/{searchType}/search` that looks odd and I'm pretty sure its wrong. just add `/affiliations/**` and `/publications/**`.